web/home.go

```diff
@@ -13,8 +13,9 @@ type HomeData struct {
 
 func HomeHandler(c *gin.Context) {
 	data := HomeData{
-		Title:     defaultLayoutData.Title,
-		Paragraph: "This is the home page",
+		Title: defaultLayoutData.Title,
+		Paragraph: `An open cloud-native web console improving on the
```

Comment:
I find this `on` from `...console improving on the...` out of place there

Reply:
I think both are right, but since your suggestion has two characters less... I agree :-p